### DISCUSSION TITLE
HS200P3 (Hardware v 5.26) Unable to Discover

### DISCUSSION THREAD

**Discussion by blancmik** [plasticrake/homebridge-tplink-smarthome#348] [General]
I have been running this plugin with HS200 Wi-Fi switches for a few years with great success. I decided to add a few additional switches and received the HS200P3 (Hardware version 5.26) from Amazon.  I cannot get them to discover or show up even if I manually add the IP address (which is how my other 7 switches work)

- My original 7 switches are hardware v5.00, running firmware 1.0.11
- New switches are hardware v5.26, firmware 1.02

I should note that these newer stitches within the Kasa app have additional configuration options, so something is certainly different.

**Comment by ctrlaltdel17** [comment]
I have the same issue as well. It looks like the new H/W 5.26 switches with this firmware don't have port 9999 open which is what the plugin is looking for. I'm really hoping a fix can be found for this.

**Reply by DustonF** [reply]
I guess I should of done some more research before purchasing lol.

**Reply by andrew1s992** [reply]
has there been any luck on getting this resolved?

**Reply by dwilasnd** [reply]
I also have this issue. HS210 just fine, HS200 can't be found

**Reply by oquatanzinwan** [reply]
Same issue here with a newly purchased HS200.  It seems that that Kasa account credentials are now required to add the switch.  I ended up having to use the "Homebridge Kasa Python" plugin and used the "Enable Credentials" setting in the plugin config.

**Comment by TheGatorFL** [comment]
Same issue here.  Thanks for posting the comments about the HW version 5.26.  Very helpful and appreciated.  I've deleted, purged, and reconfigured the plugin including manual config without any luck discovering the newly installed switches.  The older 58 switches are found without any issue.

**Reply by RabidPuffin** [reply]
Yeah, same issue as well. Sucks that these are the standard switches in the 2-packs. Can't find the 2-packs with HomeKit switches on Amazon.

**Comment by Brodie01** [comment]
Any fix for this yet???? Anyone have a workaround?

**Reply by SHWings** [reply]
Curious myself as I'm having the same issue.

**Comment by Cmanahan11** [comment]
Same issue here - Anyone solve this??

**Reply by NickRivait** [reply]
Same, Python didn’t work for me.
